inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/ConceptInstancePanel.html

```diff
@@ -26,6 +26,22 @@
 
         <!-- Instance list -->
         <div wicket:id="instances" class="flex-v-container" style="flex: 0.25 1 auto"></div>
+     	
+     	<!-- Instances in annotated text doc -->
+     	
+     	<div class="scrolling flex-content">
+          <table id="bd" class="table table-striped table-condensed">
```

Comment:
Do you need the id?

Reply:
Using Wicket, HTML elements should never have an ID element.